wiremock/bookingStubs.ts

```diff
@@ -48,7 +45,7 @@ bookingStubs.push({
   },
 })
 
-const requiredFields = getCombinations(['crn', 'name', 'expectedArrivalDate', 'expectedDepartureDate', 'keyWorkerId'])
+const requiredFields = getCombinations(['crn', 'name', 'expectedArrivalDate', 'expectedDepartureDate'])
```

Comment:
Can we remove 'name' too as we get it from the CRN now?

Reply:
Actually - whilst you're here - I think 'expectedArrivalDate' and 'expectedDepartureDate' should just be 'arrivalDate' and 'departureDate' to much the UI fields